sdcm/cluster.py

```diff
@@ -4054,6 +4054,7 @@ def start_scylla_monitoring(self, node):
         run_script = dedent("""
             cd {0.monitor_install_path}
             mkdir -p {0.monitoring_data_dir}
+            ./kill-all.sh
```

Comment:
why did you add this?

Reply:
please remove this and merge.
We need to implement "clean resources" for Docker. I will open a task for that